django_mri/filters/scan_filter.py

```diff
@@ -39,5 +39,6 @@ class Meta:
             # "spatial_resolution",
             "institution_name",
             "is_updated_from_dicom",
+            "dicom__id",
```

Comment:
It's being used in order to determine whether a particular DICOM series has a corresponding Scan instance. It might be alright to just switch it to `"dicom"` with a HyperlinkedRelatedField definition and query by url, but it also shouldn't make much of a difference.